release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1277,6 +1277,47 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
 [id="ocp-4-17-management-console-bug-fixes_{context}"]
 ==== Management Console
 
+* Previously, a warning was not provided when you were on a Google Cloud Platform (GCP) cluster that supports GCP's Workload Identity Management, and that the Operator supports it. With this release, logic was added support for GCP Workload Identity and Federated Identity Operator installs, so now you are alerted when you are on a GCP cluster. (link:https://issues.redhat.com/browse/OCPBUGS-38591[*OCPBUGS-38591*])
+
+* Previously, the version number text in the *Updates* graph on the *Cluster Settings* page appeared as black text on a dark background when using Firefox in dark mode. With this update, the text appears as white text. (link:https://issues.redhat.com/browse/OCPBUGS-38427[*OCPBUGS-38427*])
+
+* Previously, the *Cluster overview* page included a `View all steps in documentation` link that resulted in a 404 error for ROSA and OSD clusters. With this update, the link does not appear for ROSA and OSD clusters. (link:https://issues.redhat.com/browse/OCPBUGS-37054[*OCPBUGS-37054*])
+
+* Previously, dynamic plugins using PatternFly 4 were referencing variables that are not available in {product-title} 4.15 and later. This was causing contrast issues for ACM in dark mode. With this update, older chart styles are now available to support PatternFly 4 charts used by dynamic plugins. (link:https://issues.redhat.com/browse/OCPBUGS-36816[*OCPBUGS-36816*])
+
+* Previously, when the `Display Admission Webhook` warning implementation presented issues with some incorrect code. With this update, the unnecessary warning message has been removed. (link:https://issues.redhat.com/browse/OCPBUGS-35940[*OCPBUGS-35940*])
+
+* Previously, the global sync lock that applied to all HTTP-serving spawned gorotines with a sync lock that is specific to each of the refresh tokens. With this update, the global refresh sync lock on a cluster with an external OIDC environment was replaced with per-refresh token one. As a result, refresh token performance is improved by 30 to 50%. (link:https://issues.redhat.com/browse/OCPBUGS-35080[*OCPBUGS-35080*])
```

Comment:
Should this be: "Previously, the global sync lock that applied to all HTTP-servers..."?

Also, just want to make sure _gorotines_ is spelled correctly. Should it be _goroutines_?

Reply:
```suggestion
* Previously, the global sync lock that applied to all HTTP-serving spawned gorotines with a sync lock that is specific to each of the refresh tokens. With this update, the global refresh sync lock on a cluster with an external OIDC environment was replaced with a per-refresh token one. As a result, refresh token performance is improved by 30 to 50%. (link:https://issues.redhat.com/browse/OCPBUGS-35080[*OCPBUGS-35080*])
```

Reply:
Last thing, I'm not sure what "one" refers to in "per-refresh token one." Do you mean "per-refresh token sync"? If so, you could reword this sentence something like, "With this update, the global refresh sync lock on a cluster with an external OIDC environment was replaced with a sync that refreshes for each token."